.env.example

```diff
@@ -12,3 +12,6 @@ VUE_APP_BASE_URL_YOUTUBE=''
 
 ### setting up OTP authentication
 VUE_APP_OTP_SERVICE=''
+
+### sentry settings
+VUE_APP_SENTRY_DSN=''
```

Comment:
Is this only for production? Or does it work in staging as well?

Reply:
If the value is not empty/null, this will work. Otherwise, it doesn't. If needed, we can just add an env variable to staging and set this value. It'll start working.

Reply:
@sudheshna-donthineni based on discussion with Pritam, I've added a check in `main.js` to enable Sentry logs only on production. Please have a look.